pytorch_lightning/callbacks/quantization.py

```diff
@@ -194,6 +254,12 @@ def on_fit_start(self, trainer, pl_module):
         elif isinstance(self._qconfig, QConfig):
             pl_module.qconfig = self._qconfig
 
+        # Disable qconfig for any modules the user requested to skip
+        if self._check_feasible_skip(pl_module):
```

Comment:
I think the `_check_feasible_skip` is not required there and could be done directly over the iteration.

if self.modules_to_skip:
   for m in self.modules_to_skip:
          module = _recursive_getattr(pl_module, m)
          if not module:
                    raise MisconfigurationException(f'{m} is not a model attribute, cannot skip quantization')
          elif not isinstance(module, Module):
               raise MisconfigurationException(f'{m} is not a `nn.Module`, cannot skip quantization')
          module.qconfig = None